src/farcaster/types.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Frame address type to determine if the client is requesingt  verified addresses or a custody address
+ *  Note: exported as public Type
+ */
+
+export enum FarcasterAddressType {
+  VerifiedAddresses,
+  CustodyAddress,
+}
+
+/**
+ * GetFarcasterUserAddressesRequest
```

Comment:
From where the developer will get this? Like from a Frame is a message, in this case do we have an idea how this request is shaped?